index.html

```diff
@@ -7,320 +7,466 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>My Portfolio</title>
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/57d41c84f5.js" crossorigin="anonymous"></script>
 
 
 </head>
 
+
 <body>
   <header class="tool_bar">
 
     <a class="Logo" href="">
       <h2>My Logo</h2>
     </a>
 
-
     <nav>
       <a href=""> <img class="union" src="images/Union.png" alt=""></a>
     </nav>
+    <ul class="desk_nav">
+      <li>
+        <a href="">Portfolio</a>
+      </li>
+      <li>
+        <a href="">About</a>
+      </li>
+      <li>
+        <a href="">Contact</a>
+      </li>
+    </ul>
 
   </header>
 
   <section class="headline-sec">
-
-    <h1 class="greetings">I’m Maurice <br>
-      Glad to see <br> you!</h1>
+    <div class="headline_container">
+      <h1 class="greetings">I’m Maurice <br>
+        Glad to see you!</h1>
 
 
-    <p class="text2">
-      Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work
-      and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-    </p>
+      <p class="text2">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
 
 
-    <button>
-      <h2 class="connect btn_effect">LET’S CONNECT</h2>
-    </button>
-
-    <div class="media-icon">
-      <ul class="media-icon">
-        <li><a href=""><img src="images/Vector.png" alt=""></a></li>
-        <li><a href=""></a><img src="images/Linkedin icon.png" alt=""></li>
-        <li><a href=""></a><img src="images/Vector (1).png" alt=""></li>
-        <li><a href=""></a><img src="images/Vector (2).png" alt=""></li>
-        <li><a href=""></a><img src="images/Vector-last.png" alt=""></li>
-
+      <button>
+        <h2 class="connect btn_effect">LET’S CONNECT</h2>
+      </button>
+      <ul class="media-icon ">
+        <li><a href=""><img src="Twitter.png" alt=""></a></li>
+        <li><a href=""><img src="Linkedin icon.png" alt=""></a></li>
+        <li><a href=""><img src="M.png" alt=""></a></li>
```

Comment:
- [ ] Kindly make sure this area looks like the Figma design, by making sure the social icons start at the same point with the text and the "let's connect" title has the same font height and paddings as shown in the Figma design.
Your design:
![Screenshot from 2023-03-04 23-32-27](https://user-images.githubusercontent.com/97834160/222929839-c7ae9bd1-db50-4229-9f0c-84c3e43068d8.png)

Figma:
![Screenshot from 2023-03-04 23-34-20](https://user-images.githubusercontent.com/97834160/222929804-e3276560-88ee-400c-bf82-be50980125d3.png)